packages/create-vite/src/index.ts

```diff
@@ -150,6 +150,12 @@ const FRAMEWORKS: Framework[] = [
         customCommand:
           'npm create -- tsrouter-app@latest TARGET_DIR --framework react --interactive',
       },
+      {
+        name: 'redwoodsdk-standard',
+        display: 'RedwoodSDK ↗',
+        color: red,
+        customCommand: 'npx degit redwoodjs/sdk/starters/standard TARGET_DIR',
```

Comment:
```suggestion
        customCommand: 'npm exec degit redwoodjs/sdk/starters/standard TARGET_DIR',
```
Let's use `npm exec` instead as we have some special handling for it.
https://github.com/vitejs/vite/blob/b7ad80e6da515fa33bc0d453ec8092d45cec1dce/packages/create-vite/src/index.ts#L646-L660